src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/Tabs/virtual.html.twig

```diff
@@ -0,0 +1,31 @@
+{#**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ *#}
+
+{% form_theme productForm.virtual_product_file '@PrestaShop/Admin/Sell/Catalog/Product/Form/virtual_product_file_form_theme.html.twig' %}
+
+<div role="tabpanel" class="form-contenttab tab-pane" id="virtual-product-tab">
```

Comment:
So now virtual file has its own dedicated tab? Instead of being part of the quantities/stock tab like it used to?

Reply:
I think it makes sense.
This way it is the same as in a view - virtual product tab, combinations tab, pack tab, stocks tab (instead of having everything mixed up in the "stocks" tab which is not actually a stock tab anymore, but more of a "mixed all types tab"

Reply:
I get what you said, it kind of makes sense Or it would if each type really had their own dedicated tab, but it feels like only Combinations has a true tab

Packs are handled in Basic tab (at least for now), and virtual is still mixed with stock (even with your modif), and since it only has a File selection component I don't think it's worth having a single tab to handle this, so I guess it's bound to be "shared" with other info in a common tab